tasks.py.jinja

```diff
@@ -95,6 +97,12 @@ def write_code_workspace_file(c, cw_path=None):
                 "configurations": ["Attach Python debugger to running container"],
                 "preLaunchTask": "Start Odoo in debug mode",
             },
+            {
+                "name": "Run and debug Odoo tests",
+                "configurations": ["Attach Python debugger to running container"],
+                "preLaunchTask": "Run Odoo Tests in debug mode",
+                "internalConsoleOptions": "openOnSessionStart",
```

Comment:
I wanted to include a `postDebugTask` here to automatically `Start Odoo` (restart in normal mode without debug) when you finish the tests, but VSCode `compounds` doesn't support it, and when I added it in the `Attach Python debugger to running container` configuration, I got an error saying I couldn't call a task from a different workspace.
I ended up leaving this like so, but it would be nice to have that feature

Reply:
Why starting Odoo? If you launch this task, you don't want to end up with an started Odoo. You just want to run tests.

Reply:
Ok.
It's just that, if you are used to running the tests with the docker-compose command, you are normally always running an Odoo instance, and then run another for the tests. So, when the tests finish, you end up with a started Odoo.
This way, if you had a running Odoo instance, this task needs to stop it and replace it with a new one for the tests, so you end up with a stopped Odoo. But I guess that is not a problem anyway...

Reply:
No, I don't start Odoo "normally" when running tests. Or maybe yes, but I think it doesn't matter. This should be independent from that. Or there are technical limitations to this?

Reply:
It is difficult to make it independent form that, because we need to map the debugger server port to the `odoo-proxy`, and when you run a container separately (with `docker-compose run --rm odoo --test-enable [...]`) it causes issues with the docker private network and the debugger port is not mapped, so we need to run it in the same Odoo container... That's why it can't be independent from what you had running.

Reply:
OK, but there's a problem with that: when you run Odoo, you do it with several workers, and tests are not fine with workers.

Reply:
Yes, when we run the tests in this case, we are relaunching the container replacing the Odoo start command, so we wouldn't have that problem...
Is that what you meant?

Reply:
I suppose that